### PR TITLE
amam/update_self_exclusion

### PR DESCRIPTION
**Pull request by mustofa-binary**
- Update note for CR self exclusion on responsible trading page

### CODE REVIEW

File: src/javascript/app/pages/user/account/settings/self_exclusion.js [hunk 1]
```diff
@@ -277,19 +277,31 @@ const SelfExclusion = (() => {
...
             }
-
-            if ('timeout_until' in data || 'exclude_until' in data) {
+            if (is_svg) {
                 Dialog.confirm({
-                    id               : 'timeout_until_dialog',
-                    localized_message: localize('When you click "OK" you will be excluded from trading on the site until the selected date.'),
+                    id               : 'self_exclusion_dialog',
+                    localized_title  : localize('Confirm changes'),
+                    localized_message: localize('We’ll update your limits. Click <strong>Agree and accept</strong> to acknowledge that you are fully responsible for your actions, and we are not liable for any addiction or loss.'),
```
Comment (negarn) [inline src/javascript/app/pages/user/account/settings/self_exclusion.js:289]: please use placeholders for `<strong>` tags

File: src/sass/static/pages.scss [hunk 2]
```diff
@@ -1702,3 +1702,7 @@ body #not_authenticated_financial ul.checked > li {
         visibility: visible;
     }
 }
+
+#self_exclusion_dialog #dialog_content {
+    max-width: 496px;
+}
```
Comment (negarn) [inline src/sass/static/pages.scss:1708]: missing new line at end of file

File: src/templates/app/user/security/self_exclusion.jsx [hunk 3]
```diff
@@ -10,10 +10,10 @@ const SelfExclusion = () => (
...
+                <p>{it.L('Online trading can be addictive. This self-exclusion page is where you manage your online trading activity and exercise [_1]responsible trading[_2].', `<a href="${it.url_for('responsible-trading')}">`, '</a>')}</p>
             </div>
-            <p>{it.L('You may also instruct us to exclude you from trading on [_1] for a specific period of time. This self-exclusion date cannot be amended once it is set.', it.website_name)}</p>
-
+            <p data-show='-eucountry'>{it.L('Here, you can set and adjust the amount of money and time you spend trading on SmartTrader, WebTrader, and Binary Bot. Setting your limits is optional and you can adjust them at any time. If you don’t wish to set a specific limit, leave the field blank.')}</p>
+            <p data-show='-eucountry'>{it.L('You can also decide to exclude yourself entirely from our website for a specified duration. Once the self-exclusion period has ended, you can either extend it further or resume trading immediately. If you wish to reduce or remove the self-exclusion period, contact our [_1]Customer Support[_2].', `<a href="${it.url_for('contact')}">`, '</a>')}</p>
```
Comment (negarn) [inline src/templates/app/user/security/self_exclusion.jsx:16]: how come we are showing 'If you wish to reduce or remove the self-exclusion period, contact our [_1]Customer Support[_2]' to SVG? I thought the point is that they can remove or reduce it on their own?

Comment (mustofa-binary) [reply src/templates/app/user/security/self_exclusion.jsx:16]: you're right, the copy and zeplin need to be updated as well

**Review by zaki-binary — APPROVED** [review]

**Pull request merged**